eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java

```diff
@@ -179,7 +180,7 @@ private static String run(
      */
     private static void compileJava(final Path generated, final Path classes) {
         new Jaxec(
-            SnippetTestCase.jdkExecutable("javac"),
+            new Jhome().path("bin/javac").toString(),
```

Comment:
@yegor256 What do you think if we add `javac()` and `java()` methods to `Jhome`. Then, instead of `new Jhome().path("bin/javac")` we will need to write only `new Jhome().javac()`?

Reply:
@yegor256 Created a separate issue for that: https://github.com/yegor256/jhome/issues/5